.github/ISSUE_TEMPLATE/Bug.md

```diff
@@ -32,22 +32,29 @@ Output of `restic version`
 --------------------------
 
 
-How did you run restic exactly?
--------------------------------
+
+What backend/server/service did you use to store the repository?
```

Comment:
I'd probably skip the `server/` part here, but that's just me.

Reply:
The server was part of the old section title. But I agree, that it's somewhat redundant.